plugins/frontend/backstage-plugin-github-pull-requests/src/components/useGithubSearchPullRequest.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2021 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { useApi, githubAuthApiRef } from '@backstage/core-plugin-api';
+import { Octokit } from '@octokit/rest';
+import { useAsync } from 'react-use';
+import {
+  GetSearchPullRequestsResponseType,
+  GithubSearchPullRequestsDataItem,
+} from '../types';
+
+export const useGithubSearchPullRequest = (query: string) => {
+  const githubAuthApi = useApi(githubAuthApiRef);
+
+  return useAsync(async (): Promise<GithubSearchPullRequestsDataItem[]> => {
+    const token = await githubAuthApi.getAccessToken(['repo']);
+
+    const pullRequestResponse: GetSearchPullRequestsResponseType =
+      await new Octokit({
+        auth: token,
+        baseUrl: 'https://api.github.com',
```

Comment:
Here as well URL.
Check `packages/integration/config.d.ts:89` on Backstage repo. The URL value is exposed to FE so we can use it from the config directly and default to api.github.com if there's no value. 
More info: https://backstage.io/docs/integrations/github/locations

Reply:
👍🏿